test/integration/scheduler_perf/util.go

```diff
@@ -282,3 +323,51 @@ func (tc *throughputCollector) collect() []DataItem {
 
 	return []DataItem{throughputSummary}
 }
+
+// getExtensionPointDurationHist return a labeld Histogram for `scheduler_framework_extension_point_duration_seconds` metric.
+// TODO: Consolidate this method with `testutil.GetHistogramFromGatherer` to remove the duplicated code.
+func getExtensionPointDurationHist(gatherer metrics.Gatherer, metricName string) ([]labeledHist, error) {
+	var metricFamily *dto.MetricFamily
+	var hists []labeledHist
+
+	m, err := gatherer.Gather()
+	if err != nil {
+		return nil, err
+	}
+	for _, mFamily := range m {
+		if mFamily.GetName() == metricName {
+			metricFamily = mFamily
+			break
+		}
+	}
+
+	if metricFamily == nil {
+		return nil, fmt.Errorf("metric %q not found", metricName)
+	}
+
+	if metricFamily.GetMetric() == nil {
+		return nil, fmt.Errorf("metric %q is empty", metricName)
+	}
+
+	if len(metricFamily.GetMetric()) == 0 {
+		return nil, fmt.Errorf("metric %q is empty", metricName)
+	}
+
+	metrics := metricFamily.GetMetric()
+	for _, metric := range metrics {
+		for _, label := range metric.GetLabel() {
+			if label.GetName() == "extension_point" {
+				if extentionsPoints[label.GetValue()] {
```

Comment:
do we plan to disable this for any extension point? if not, I think we can simplify this and remove the extentionsPoints map altogether. IMO having that map only adds maintenance overhead (if/when we add a new extension point for example).

Reply:
I incline to agreeing with you.